internal/dataplane/parser/translate_httproute_test.go

```diff
@@ -18,24 +18,27 @@ import (
 	"github.com/kong/kubernetes-ingress-controller/v2/internal/util"
 )
 
-// httprouteGVK is the GVK for HTTPRoutes, needed in unit tests because
-// we have to manually initialize objects that aren't retrieved from the
-// Kubernetes API.
-var httprouteGVK = schema.GroupVersionKind{
-	Group:   "gateway.networking.k8s.io",
-	Version: "v1alpha2",
-	Kind:    "HTTPRoute",
-}
+var (
+	// httprouteGVK is the GVK for HTTPRoutes, needed in unit tests because
+	// we have to manually initialize objects that aren't retrieved from the
+	// Kubernetes API.
+	httprouteGVK = schema.GroupVersionKind{
+		Group:   "gateway.networking.k8s.io",
+		Version: "v1alpha2",
+		Kind:    "HTTPRoute",
+	}
+
+	pathMatchPrefix = gatewayv1alpha2.PathMatchPathPrefix
+	pathMatchRegex  = gatewayv1alpha2.PathMatchRegularExpression
+	pathMatchExact  = gatewayv1alpha2.PathMatchExact
+	queryMatchExact = gatewayv1alpha2.QueryParamMatchExact
```

Comment:
[Aren't those `const`s in the gateway package](https://github.com/kubernetes-sigs/gateway-api/blob/v0.5.0/apis/v1alpha2/httproute_types.go#L258-L282)? Why make them `var`s here? Let's reuse the constness of those and declare them as such here

Reply:
We need references to them. Is there some better way to handle that? consts will get us: 

```
vet: internal/dataplane/parser/translate_httproute_test.go:226:17: invalid operation: cannot take address of pathMatchPrefix (constant "PathPrefix" of type "sigs.k8s.io/gateway-api/apis/v1beta1".PathMatchType)
```
https://github.com/kubernetes-sigs/gateway-api/commit/9138381f8bb34a0fec4d3671cbddf628bc5edb0b will be available later, but not in 0.5.0.